### PR TITLE
Update to CalDigit TS3 Plus docking station compatibility on Adder WS…

### PR DESCRIPTION
**Pull request by slauzon**
Update to CalDigit TS3 Plus docking station compatibility and issues associated with the Adder WS (addw1) laptop.

### CODE REVIEW

File: content/docking-station.md [hunk 1]
```diff
@@ -59,6 +60,9 @@ Community members have reported that the following docks work with our products:
 - [Anker A8392](https://us.anker.com/products/a8392) [[community-tested](https://github.com/system76/docs/pull/797) on an Intel system] <sup>1</sup>
 - [CalDigit TS3 Plus](https://www.caldigit.com/ts3-plus/) [[community-tested](https://github.com/system76/docs/pull/417) on an Intel system] <sup>1</sup>
   - Downstream (passthrough) Thunderbolt 3 port not tested.
+  - May not be able to wake system using peripherals connected via dock.
+  - These issues can be potentially addressed by changing the Thunderbolt Security Mode to "None" in the BIOS, though this is not recommended.
+    - Displays sometimes don't wake up from sleep until dock is re-plugged.
```
Comment (jacobgkau) [inline content/docking-station.md:65]: I'm a little confused by the indentation here-- is it the displays issue that can be resolved by changing the security level? If so, I would swap the indentations so the workaround is a sub-bullet of the issue.

**Comment by slauzon** [comment]
@jacobgkau Committed changes to address your concerns.  Thanks for the review.

**Comment by jacobgkau** [comment]
@slauzon Thank you. Can you clarify what "potentially" means in the workaround? I assume you have an addw1 and a TS3 Plus dock, does the workaround work or does it not work/have you not tried it? Does it depend on something else?

**Comment by slauzon** [comment]
@jacobgkau The workaround appeared to work at first, but now I can't get the monitor to wake properly again.  I will adjust my commit accordingly.  FYI I have a ticket in with System76 support and will be opening one soon with CalDigit to see if they have any ideas.

**Review by jacobgkau — APPROVED** [review]

**Pull request merged**